lib/spack/spack/util/web.py

```diff
@@ -106,7 +106,8 @@ def read_from_url(url, accept_content_type=None):
         else:
             # User has explicitly indicated that they do not want SSL
             # verification.
-            context = ssl._create_unverified_context()
+            if not __UNABLE_TO_VERIFY_SSL:
```

Comment:
This is here because the SSLContext object does not exist prior to Python 3.2, __UNABLE_TO_VERIFY_SSL checks whether Python supports ssl contexts.[ Above this, in line 100, it is used to similar effect](https://github.com/spack/spack/blob/d9bf7b392eb3d1745a4b7a36e33b5288ebfb4f76/lib/spack/spack/util/web.py#L100). If __UNABLE_TO_VERIFY_SSL is true, no context is created even if the user wants one.

Reply:
Can you add a comment about this?